tgui/packages/tgui/components/DepartmentPane.tsx

```diff
@@ -0,0 +1,73 @@
+import { Color } from 'common/color';
+import { SFC } from 'inferno';
+import { Box } from './Box';
+import { Stack } from './Stack';
+
+// The cost of flexibility and prettiness.
+export const DepartmentEntry: SFC<{
+  style?;
+  titleStyle?;
+  textStyle?;
+  title?;
+  titleContents?;
+  titleSubtext?;
+}> = (props) => {
+  return (
+    <Box ClassName="Section" style={props.style}>
+      <Box class="Section__title" style={props.titleStyle} minHeight="3.4rem">
+        <Box class="Section__titleText" style={props.textStyle}>
+          {props.title}
+          {props.titleContents}
+        </Box>
+        {props.titleSubtext}
+        <br style={{ 'clear': 'both' }} />
```

Comment:
@stylemistake Should we really be using <br /> instead of stacks? I don't think we do this anywhere

Reply:
Nope, I even think RimiNosha should reuse Section__buttons to align things to the right, I kinda find that newline between the title and text below ugly

Reply:
Ah, nice to know about `Section__buttons`, though in my testing, not having the newline causes weird clipping. Might be due to the float though.

Reply:
Yup, this looks better overall, and is 900% less jank!